### PR TITLE
Add module outputs

### PR DESCRIPTION
**Pull request by Fodoj**
As proposed here: https://github.com/hashicorp/terraform/issues/1940 
Instead of implementing something like `terraform output "path.to.module output_name"` I've used approach from `taint` command: using `-module` argument.

### CODE REVIEW

**Comment by Fodoj** [comment]
Any opinion on this one? I'd love to see it merged, because we have a strong need for showing nested outputs.

**Comment by Fodoj** [comment]
@mitchellh  Any opinion on this one? I'd love to see it merged, because we have a strong need for showing nested outputs.

File: command/output.go [hunk 1]
```diff
@@ -38,15 +41,34 @@ func (c *OutputCommand) Run(args []string) int {
...
+	// Get the proper module we want to get outputs for
+	modPath := strings.Split(module, ".")
+
 	state := stateStore.State()
-	if state.Empty() || len(state.RootModule().Outputs) == 0 {
+	mod := state.ModuleByPath(modPath)
+
+	if mod == nil {
+		c.Ui.Error(fmt.Sprintf(
+			"The module %s could not be found. There is nothing to taint.",
```
Comment (phinze) [inline command/output.go]: `s/nothing to taint/nothing to output/`

**Comment by phinze** [comment]
One typo, but otherwise this LGTM! (I'll just merge and fix the typo on master.)

**Comment by Fodoj** [comment]
Thanks!

**Pull request merged**